source/includes/v2/_odsql.md

```diff
@@ -134,17 +134,66 @@ Supported geometry expressions are:
 
 A scalar function can be used in [select arithmetic expressions](#arithmetic-select-expression), [filter expressions](#filter-expression) or in the [group by clause](#group-by-clause).
 
-Function|Parameters|Description|Limitation
---------|-----------|----------|----------
-`length`|string literal or string field literal|Returns the number of characters|
-`now`|no parameter|Returns the current date|Only works on filter expressions
-`year`|date field literal|Returns the year of a date|
-`month`|date field literal|Returns the month of a date|
-`day`|date field literal|Returns the day of a date|
-`hour`|date field literal|Returns the hour of a date|
-`minute`|date field literal|Returns the minute of a date|
-`second`|date field literal|Returns the second of a date|
-`date_format`|date field literal|Returns a formatted date|
+Function|Parameters|Description
+--------|-----------|----------
+`length`|string literal or string field literal|Returns the number of characters
+`now`|optional named parameters (see below)|Returns the current date
+`year`|date field literal|Returns the year of a date
+`month`|date field literal|Returns the month of a date
+`day`|date field literal|Returns the day of a date
+`hour`|date field literal|Returns the hour of a date
+`minute`|date field literal|Returns the minute of a date
+`second`|date field literal|Returns the second of a date
+`date_format`|date field literal|Returns a formatted date
+
+#### Parameters to the now() function
+
+> Examples, assuming the current date time is 2021-05-06 12:34:55.450500+00:00, which is a thursday
+
+```sql
+now() -- Returns '2021-05-06T12:34:55.450500+00:00'
+now(year=2000) -- Sets the year component to return '2000-05-06T12:34:55.450500+00:00'
+now(year=2001, months=-1) -- Sets the year to 2001 and substract 1 month to return '2000-04-06T12:34:55.450500+00:00'
```

Comment:
There's a typo here: "substract" --> subtract.
```suggestion
now(year=2001, months=-1) -- Sets the year to 2001 and subtract 1 month to return '2000-04-06T12:34:55.450500+00:00'
```

Reply:
! I did not know it was a mistake ! Thanks